docs/framework/migration-guide/versions-and-dependencies.md

```diff
@@ -49,6 +50,24 @@ Jump to:
 - [.NET Framework 1.1](#net-framework-11)
 - [.NET Framework 1.0](#net-framework-10)
 
+### .NET Framework 4.8.1
+
+- [New features](../whats-new/index.md#whats-new-in-net-framework-481)
+- [New in accessibility](../whats-new/whats-new-in-accessibility.md#whats-new-in-accessibility-in-net-framework-481)
```

Comment:
```suggestion
- New accessibility features
```

Reply:
Since the bookmark doesn't exist yet, we either have to leave the link off for now or leave the bookmark off.